src/ppx/application.cpp

```diff
@@ -1219,6 +1238,38 @@ int Application::Run(int argc, char** argv)
             mAverageFrameTime = static_cast<float>(mTimer.MillisSinceStart() / mFrameCount);
         }
 
+        // Update default metrics in the current metrics run.
+        if (mSettings.useMetrics && mMetrics.pCurrentRun != nullptr) {
+            PPX_ASSERT_NULL_ARG(mMetrics.pCpuFrameTime);
```

Comment:
Should we move this to a helper function? Run() is pretty long

Reply:
I moved the metrics system update code into its own function.